rfc-018-our-decision-making-process-is-public.md

```diff
@@ -0,0 +1,69 @@
+# Our decision making process is public
+
+## Summary
+
+Our process of making decisions and the decisions themselves MAY be read by
+all staff and members of the public.
+
+This topic has previously been [debated as an issue](https://github.com/dxw/tech-team-rfcs/issues/2)
+on this repository. The decision to take no action should still be persisted
+as a formal decision so has become an RFC.
+
+## Problem
+
+This repository started as public by default but do we agree that this is right
+in this case?
+
+Reasons for:
+
+- dxw values include being honest and we agree with the
+[principle of making things open as it makes things better](https://www.gov.uk/guidance/government-design-principles#make-things-open-it-makes-things-better)
+- [the dxw playbook is public](http://playbook.dxw.com) and has not caused us
+concern or prevented us from iterating it
+- others can learn from us and spend more their time making things better for
+users
+- it allows us to share our thinking and rationale on our technical decisions,
+doing so publicly as well as privately can increase trust and empathy with more
+people
+- with the potential for the world watching the bar is raised
+  being open from the start ensures that there is less risk to us should a
+  future team decide to make it public
+- most of the application code we write lives in open source repositories where
+our work and decision making process can already be viewed publicly
+
+Reasons against:
+
+- parts of our discussion make have to include commercially sensitive
+ information eg. incident reports
+- as the public can read the team may feel too uncomfortable to contribute to
+ discussions making it harder to talk with candour
+
+It's difficult to change our mind if we start from a private repository.
+
+Contributors are aware of the context in which they add their comments. If
+ they make comments under the context of a private repository which then
+ becomes public, there is a potential risk for us to make content that was
+ not intended or written in a way appropriate for public consumption.
+ Therefore starting from a private repository and later deciding to make
+ it a public one will be incredibly difficult to do safely. By starting with
+ a public repository however, we always have the safe and easy option of
+  making private open to us.
+
+## Proposal
+
+We SHOULD NOT make this repository private.
+
+We SHOULD promote sensitive discussions from GitHub into our regular
+ tech-forums. Here we will discuss the issue in a private forum and we MUST
+  publish a safe summary of the discussion to allow voting to continue as
+   normal with adequate reference.
+
+We SHOULD have a retrospectives on how the team have found making decisions
+ in public.
```

Comment:
How frequently? Is there a maximum time that we feel comfortable setting? At least once every 6 months?

Reply:
I agree we should be thinking about time so that it is actionable however I'm not sure the RFC is responsible for capturing how we implement this. Perhaps a next step should be to schedule one for 6 months time.